taxonomy-editor-frontend/src/pages/editentry/AccumulateEntryInfo.jsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Typography } from "@mui/material";
+import ListAllEntryProperties from "./ListAllEntryProperties";
+import ListTranslations from "./ListTranslations";
+
+/**
+ * Parent component used for rendering sub-components
+ * Sub-components are used to render info for an "entry"
+ */
+
+const AccumulateEntryInfo = ({ nodeObject, setNodeObject, originalNodeObject }) => {
```

Comment:
maybe this component could be used directly, I mean, remove this component and pass the things to the parent